src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -395,6 +395,9 @@ export default {
 		},
 
 		internalAbsolutePath() {
+			if (!this.file.path) {
+				return ''
+			}
```

Comment:
Same, in which case it is empty?

Reply:
when file is in process of upload -> not at the server => doesn't have a path